### DISCUSSION TITLE
What is the meaning of "new version" in update method?

### DISCUSSION THREAD

**Discussion by nepgya2803** [pnp/pnpcore#1291] [Q&A] [answered]
I'm a bit confused when it comes to these update methods like 
**Update, UpdateAsync** : Regular update, this will result in a new version being created and the modified and editor list item fields will be updated

**SystemUpdate, SystemUpdateAsync**: Updates the item without creating a new version and without updating the modified and editor list item fields

**UpdateOverWriteVersion, UpdateOverWriteVersionAsync**: Updates the item without creating a new version and the modified and editor list item fields will be updated

What is the meaning of the term "new version" in these concepts? Does it affect when and which situation should I use these methods?

**Comment by jansenbe** [comment, marked as answer]
@ngqtrieu99 : when versioning is enabled on a list then normally each update will trigger the creation of a new version. Using these other methods the new version is not created as part of the update.
